src/lib/viewers/image/ImageViewer.js

```diff
@@ -81,6 +81,11 @@ class ImageViewer extends ImageBaseViewer {
 
         this.currentRotationAngle = 0;
 
+        const fileName = this.options.file && this.options.file.name && this.options.file.name.split('.')[0];
```

Comment:
Can we use `lodash/get` to simplify our property accessors here? Can we include the extension in the alt text, as well, to further simplify?